pkg/attestation/scorecard/scorecard.go

```diff
@@ -0,0 +1,182 @@
+// Copyright 2022 The Witness Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard
+
+import (
+	"crypto"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/testifysec/witness/pkg/attestation"
+	"github.com/testifysec/witness/pkg/cryptoutil"
+)
+
+const (
+	Name      = "scorecard"
+	Type      = "https://witness.testifysec.com/attestations/scorecard/v0.1"
+	RunType   = attestation.PostRunType
+	mimeTypes = "text/plain, application/json"
```

Comment:
Same comment as the sarif attestor -- just define the array here instead of splitting the string